octopus-agent/src/main/java/octopus/teamcity/agent/OctopusOsUtils.java

```diff
@@ -29,16 +29,17 @@
     private static final Logger LOGGER = Loggers.AGENT;
 
     public static Boolean CanRunOcto(@NotNull BuildAgentConfiguration agentConfiguration){
-        if (agentConfiguration.getSystemInfo().isUnix()) {
+        if (agentConfiguration.getSystemInfo().isUnix() || agentConfiguration.getSystemInfo().isMac()) {
+            String os = agentConfiguration.getSystemInfo().isUnix() ? "Unix" : "OSX";
```

Comment:
Not sure if it'd be better to have `Mac` here rather than `OSX` 🤔

Reply:
They renamed it to `macOS` a while back... Not sure if that helps or just makes it more confusing :)